### PR TITLE
Updates the Docs

### PR DESCRIPTION
**Pull request by hotzenklotz**
I updated the docs very slightly:

- move skeleton and volume annotations into their own seperate, first-order pages
- refined description of merger mode
- high-level overview of mapping support
- slight renaming of some title (feedback appreciated)

TODO:
- [x] look into way of getting videos to work

### URL of deployed dev instance (used for testing):
- https://docs.webknossos.org/v/doc_update/

### Steps to test:
- Read the new docs

------
- [x] Updated [documentation](../blob/master/docs) if applicable
- [ ] Ready for review

### CODE REVIEW

**Comment by hotzenklotz** [comment]
@philippotto Can you please help here. I would like to include a short example of a mapping file / schema. See my `TODO` comment. I am also unsure on where the mapping files need to be in the file structure in order to be detected. Some refinement of my text is appreciated. :-)

File: docs/volume_annotation.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+"Merger mode" can be enabled in the settings under "Nodes & Trees" with the option "Enable Merger Mode". As soon as you enable it, all already existing trees will be used to form merged segments.
+
+### Mappings / On-Demand Agglomeration
+With webKnossos it is possible to apply a precomputed agglomeration file to combine over-segmentated volume annotations on demand. This eleviates the demand to materialize agglomeration results prematurely and allows researchs to apply different agglomerations to the data for experimentation. We typically produce several agglomeration results based on diffent thresholds and methods and use webKnossos to quickly review theses results on subsets of a dataset in an interactive session.
+
+Mapping files are automatically identified by webKnossos when being placed in the same directory as the data. All available mappings can be activated from a dropwdown in the "Segmentation" information pane, typically on the right-hand side of the screen. Due to their file size, mappings are fetched on demand before being applied. Users can easily switch between several mappings and webKnossos will update accordingly.
```
Comment (philippotto) [inline docs/volume_annotation.md]: typo in "dropdown"

File: docs/volume_annotation.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Mapping files are in JSON and need to follow this schema:
+```
+{
+  TODO
```
Comment (philippotto) [inline docs/volume_annotation.md]:
```
{
   "name": "astrocytes",
   "classes": [
		[
			69381,
			69445,
			138248
		], [
			138307,
			343831
		], [
			348348,
			132432,
			387433,
			338330
		]
    ]
}
```

File: docs/volume_annotation.md (hunk 1 above)
Comment (philippotto) [inline docs/volume_annotation.md]: `when being placed in a "mappings" folder within the segmentation folder`

**Pull request merged**